practice-javascript/src/js/controllers/signupController.js

```diff
@@ -0,0 +1,44 @@
+const api = "http://localhost:3000/user";
+
+const username = document.querySelector("#email");
+const password = document.querySelector("#password");
+const fname = document.querySelector("#fname");
+const lname = document.querySelector("#lname");
+const avatar = document.querySelector("#avatar");
+const phonenumber = document.querySelector("#phone");
+const btnSignUp = document.querySelector(".sign-up__btn");
+
+btnSignUp.addEventListener("click", (e) => {
+  e.preventDefault();
+  if (
+    username.value == "" ||
+    password.value == "" ||
+    fname.value == "" ||
+    lname.value == "" ||
+    avatar.value == "" ||
+    phonenumber.value == ""
+  ) {
+    alert("not empty");
```

Comment:
What field can't be empty? If you show the alert like this, user don't know how to fill the right value